pkgs/build-support/cc-wrapper/ld-solaris-wrapper.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!@shell@
```

Comment:
`@shell@` here would be `stdenv.shell`, yes. There's a builder/bash procedure `substituteAll infile outfile` that substitutes all `@pattern@` with the corresponding shell parameter, so if you have `$shell = "${bash}/bin/bash"` in the environment when substitution occurs, `@shell@` is substituted for that value. From Nix, you'd do something like `substituteAll { inherit (stdenv) shell; src = ./foo.sh; }`.